Twig/DataGridExtension.php

```diff
@@ -38,131 +38,173 @@ class DataGridExtension extends \Twig_Extension
     protected $router;
 
     /**
-     * @var array
+     * @var string[]
      */
     protected $names;
 
+    /**
+     * @param \Symfony\Component\Routing\Router $router
+     */
     public function __construct($router)
     {
         $this->router = $router;
     }
 
-    public function initRuntime(\Twig_Environment $environment)
-    {
-        $this->environment = $environment;
-    }
-
     /**
-     * Returns a list of functions to add to the existing list.
-     *
-     * @return array An array of functions
+     * @return array
      */
     public function getFunctions()
     {
         return array(
-            'grid'              => new \Twig_Function_Method($this, 'getGrid', array('is_safe' => array('html'))),
-            'grid_titles'       => new \Twig_Function_Method($this, 'getGridTitles', array('is_safe' => array('html'))),
-            'grid_filters'      => new \Twig_Function_Method($this, 'getGridFilters', array('is_safe' => array('html'))),
-            'grid_rows'         => new \Twig_Function_Method($this, 'getGridItems', array('is_safe' => array('html'))),
-            'grid_pager'        => new \Twig_Function_Method($this, 'getGridPager', array('is_safe' => array('html'))),
-            'grid_actions'      => new \Twig_Function_Method($this, 'getGridActions', array('is_safe' => array('html'))),
-            'grid_url'          => new \Twig_Function_Method($this, 'getGridUrl'),
-            'grid_filter'       => new \Twig_Function_Method($this, 'getGridFilter'),
-            'grid_cell'         => new \Twig_Function_Method($this, 'getGridCell', array('is_safe' => array('html'))),
+            new \Twig_SimpleFunction(
```

Comment:
Yeah, I didn't do that because I wanted to keep it consistent. On the other hand, I also changed `array()` to `[]`...